pymc/logprob/basic.py

```diff
@@ -445,8 +445,8 @@ def conditional_logp(
 
     .. math::
 
-        \log p(Y = y, \Sigma^2 = \sigma^2) =
-            \log p(Y = y \mid \Sigma^2 = \sigma^2) + \log p(\Sigma^2 = \sigma^2)
+        \log p_{Y,s^2}(y, \sigma^2) =
```

Comment:
```suggestion
        \log p_{Y, \sigma^2}(y, s^2) =
```
For this line in particular, your original line can be correct. To keep consistent with the code and other comments, I would prefer what I just suggested